### ISSUE TITLE
Avoid cell_group_ItemType for applyOperation for a homogeneous group

### ISSUE THREAD

**Issue by DavidDureau** [arcaneframework/framework#721]
Assume we call `cell_group.applyOperation(op)` in the case where `cell_group` is a homogeneous group (ie every items have the same type (Quad4 for example)).

Arcane builds a new CellGroup called "cell_group_Quad4" which duplicates the original "cell_group".

For this particular case of homogeneous group, is it possible not to build the "cell_group_ItemType" and to use directly the original cell_group?

**Comment by grospelliergilles** [comment]
Yes it will be possible. It will be available in a couple of weeks. Before that we need to add some attributes to the mesh to know it is a structured mesh.
